doc/manual/rl-next/zzz-other.md

```diff
@@ -0,0 +1,50 @@
+---
+synopsis: Other changes
+---
+
+- [#9063](https://github.com/NixOS/nix/pull/9063): introduce `libnixflake` and move flakes-specific code there (Nix is internally composed of a set of libraries, and this better reflects the architecture wrt flakes)
+- [#10852](https://github.com/NixOS/nix/pull/10852): make Nix commands respond better to interruption - Ctrl+C in the terminal
+- [#10853](https://github.com/NixOS/nix/pull/10853): fix docs of `builtins.importNative`
```

Comment:
https://github.com/NixOS/nix/pull/10853/files

This is not what the PR is about